server/config/topic_page/poverty/USA.textproto

```diff
@@ -310,4 +332,43 @@ blocks {
     type: LINE
     title: "Percentage of households in poverty with supplemental social security income"
   }
-}
\ No newline at end of file
+}
+
+blocks {
+  title: "Education attainment"
+  stat_var_metadata {
+    stat_var: "Count_Household_HouseholderEducationalAttainmentBachelorsDegreeOrHigher_FamilyHousehold_BelowPovertyLevelInThePast12Months"
+    name: "Bachelor's Degree or higher"
+  }
+  stat_var_metadata {
+    stat_var: "Count_Household_HouseholderEducationalAttainmentHighSchoolGraduateIncludesEquivalency_FamilyHousehold_BelowPovertyLevelInThePast12Months"
+    name: "High school graduate"
+  }
+  stat_var_metadata {
+    stat_var: "Count_Household_HouseholderEducationalAttainmentLessThanHighSchoolGraduate_FamilyHousehold_BelowPovertyLevelInThePast12Months"
+    name: "Less than high school"
+  }
+  stat_var_metadata {
+    stat_var: "Count_Household_HouseholderEducationalAttainmentSomeCollegeOrAssociatesDegree_FamilyHousehold_BelowPovertyLevelInThePast12Months"
+    name: "Some college or associate's degree"
+  }
+  left_tiles {
+    type: LINE
+    title: "Education attainment for households in poverty"
+  }
+  right_tiles {
+    type: BAR
+    title: ""
+  }
+}
+
+# blocks {
```

Comment:
comment what this commented out section is for?

Reply:
done